1-E-Commerce/index.html

```diff
@@ -0,0 +1,258 @@
+                    <p class="card-description">${card.description}</p>
+                </div>
+                
+                <div class="card-footer">
+                    <div class="card-price">€ ${card.price}</div>
+    
+                        <div class="card-fav-icon">
+                            <svg class="fav ${(card.favorite) ? 'fav-red' : ''}" data-product_id="${card.id}" 
+                                width="15" height="15" viewBox="0 0 17 17" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <path data-product_id="${card.id}" fill-rule="evenodd" clip-rule="evenodd" d="M11.6386 1.82364C10.3702
+                                1.8186 9.18685 2.46096 8.50001 3.52735C7.81317 2.46096 6.62981 1.8186 5.36138
+                                1.82364C4.35178 1.82558 3.38462 2.23001 2.67417 2.94734C1.96371 3.66466 1.56861
+                                4.63566 1.57638 5.64524C1.57638 9.44384 8.50001 15.1764 8.50001 15.1764C8.50001
+                                15.1764 15.4236 9.44384 15.4236 5.64524C15.4314 4.63566 15.0363 3.66466 14.3259
+                                2.94734C13.6154 2.23001 12.6482 1.82558 11.6386 1.82364Z"
+                                stroke="#EF4444" stroke-linecap="square"/>
+                            </svg>
+                        </div>
+                </div>
+            `
+
+            return container
+        }
+
+        const populateWithItems = function (items = products) {
+            const container = document.querySelector('.products')
+
+            items.forEach(product => {
+                const node = createCardNode(product)
+
+                container.append(node);
+            })
+        }
+
+        const augmentLimit = function (augmentBy = 6) {
+            const currentLimit = JSON.parse(localStorage.getItem('limit'))
+
+            const validationPassed = typeof currentLimit === 'number' &&
+                Number.isInteger(currentLimit) &&
+                currentLimit >= 6
+
+            if (validationPassed) {
+                localStorage.setItem('limit', JSON.stringify(currentLimit + augmentBy))
+            } else {
+                localStorage.setItem('limit', '6')
+            }
+        }
+
+        const clearItems = function () {
+            const container = document.querySelector('.products')
+            container.innerHTML = ''
+        }
+
+        const sortAlphabetically = function (desc = true) {
+            clearItems()
+ 
+            products.sort((a, b) => {
+                return (desc) ? a.title.localeCompare(b.title): b.title.localeCompare(a.title);
+            })
+
+            populateWithItems(products.slice(0, 6))
+        }
+
+        const sortByPrice = function (desc = true) {
+            clearItems()
+ 
+            const order = (desc) ? 1 : -1;
+ 
+            products.sort((a, b) => (a.price > b.price) ? order : -order)
+
+            populateWithItems(products.slice(0, 6))
```

Comment:
I think you could just do this:

`products.sort((a, b) => order ? a.price - b.price : b.price - a.price)`

Reply:
Yes! But with one small difference (use the boolean`desc` instead of `order` that currently always evaluates to true)

`products.sort((a, b) => desc ? a.price - b.price : b.price - a.price)`

Reply:
Ah yes my bad haha